lib/src/cloud_firestore_mocks_base.dart

```diff
@@ -40,6 +44,18 @@ class MockFirestoreInstance extends Mock implements Firestore {
     return jsonText;
   }
 
+  void saveDocument(String path) {
```

Comment:
Consider moving all the saveDocument functionality to a class that is not exposed, so that it doesn't show up in the documentation.

Reply:
I've filed #36 to keep track of it.